plugins/tiddlywiki/googleanalytics/googleanalytics.js

```diff
@@ -1,9 +1,9 @@
-/*\
+﻿/*\
 title: $:/plugins/tiddlywiki/googleanalytics/googleanalytics.js
 type: application/javascript
 module-type: startup
 
-Runs Google Analytics with the account number in the tiddler `$:/GoogleAnalyticsAccount` and the domain name in `$:/GoogleAnalyticsDomain`
+Runs new "i s o g r a m" Google Analytics with the account number in the tiddler `$:/GoogleAnalyticsAccount` and the domain name in `$:/GoogleAnalyticsDomain`
```

Comment:
For now there are two versions of tracker. But I did not see any version reference in Google documentation. Just the strong suggest to switch to the new one.